plugins/cue_points/session_cue_point/lib/api/filters/KalturaSessionCuePointFilter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @package plugins.sessionCuePoint
+ * @subpackage api.filters
+ */
+class KalturaSessionCuePointFilter extends KalturaSessionCuePointBaseFilter
+{
+	/* (non-PHPdoc)
+ 	 * @see KalturaFilter::getCoreFilter()
+ 	 */
+	protected function getCoreFilter()
+	{
+		return new SessionCuePointFilter();
+	}
+	
+	/* (non-PHPdoc)
+	 * @see KalturaRelatedFilter::validateForResponseProfile()
+	 */
+	public function validateForResponseProfile()
```

Comment:
this should be empty like on other cue point filter types, the only one that uses isPublicEqual is annotation cue point which actually has this field on the object